src/locales/en/pages.yml

```diff
@@ -21,8 +21,15 @@ pyiNoMatch:
     - <a href="/ipv/journey/next" class="govuk-button" data-module="govuk-button"> {{ translate("buttons.next") }} </a>
     - <a href="https://signin.account.gov.uk/contact-us" class="govuk-link"> Contact the GOV.UK account team (opens in a new tab) </a>
 
-
-# Pages created from the prototype
+pyiKbvFail:
+  title: Sorry, we cannot prove your identity right now
+  content:
+    - This might be because you gave the wrong answer to some of the security questions.
+    - To keep your information safe, we cannot show which questions you answered incorrectly. 
```

Comment:
This is correct as it follows what's in figma, but I'm just wondering about the use of `cannot`. I seem to remember learning from Sabrina that we shouldn't use negative contractions like this, and it should be `can not`. I'll check in with her. We can always update it if needed.

Reply:
'Cannot' is fine, that's correct! It would be something like 'can't' that's a contraction and against our style guide 😁

Reply:
Thank you!